_pages/contributing/extensibility-customizability-guidelines.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Writing extensible and customizable code in Spartacus
```

Comment:
It should be nested in the navigation in Coding Guidelines

Reply:
Done 🙂